### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by GrtPwrflOz** [Sekai-World/sekai-viewer#393]
**Describe the bug**
 Whenever I try to use vivid bad squad Luka on VTube studio it gives me a error

**To Reproduce**
5. *
1: download the model
2: import the files in
3: select the model

*Expected behavior**

A working model like the others

**Screenshots**

![FF564DDA-C301-43AA-B748-04138BD90D94](https://user-images.githubusercontent.com/107433205/173444802-46654195-3b12-49ac-a492-e3ec1e55255e.jpeg)
![70A8A4DA-12A0-47E6-AD0E-50901E711473](https://user-images.githubusercontent.com/107433205/173444821-a2054ba4-9f9d-4a62-82bc-5f0533225f57.png)
Idk if these will work

**Browser (please complete the following information):**
OS: IPhone
Browser: safari
Version: 15.5
**Possible cause and/or solution**

Idk :(

**Comment by dnaroma** [comment]
Does it work on sekai viewer?

**Comment by GrtPwrflOz** [comment]
> Does it work on sekai viewer?

Yeah Just checked it seems to work fine on the live2d section idk why it's not working on vtube studio
